tests/detect-bytemath-04/README.md

```diff
@@ -0,0 +1,11 @@
+Description
+===========
+Test byte_match payload keyword.
```

Comment:
What's the difference between both the tests?

Reply:
I have tested (-) in one (+) in other but I think I can combine them in one and increase the number of rules?

Reply:
IMO, it's ok to combine tests `-03` and `-04` provided the rule count doesn't get "too high". 
It's also ok to keep the tests separate -- if they remain separate, I'd suggest changing the test directory name slightly to `detect-bytemath-sub-03` and `detect-bytemath-add-04`